### ISSUE TITLE
[carddav] collected addresses address book is not shown

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#409]
```
Token: 54b582aa3ab4be2dab0d0ad35c3447c0 (Don't modify this token!)
Version: 2.6 (02-11-2012)
PHP: 5.3.9-6
RCMAIL: 0.8.2
Database: mysql
SERVER: Apache
----
I.  Issue Description: I noticed that the collected addresses (which I use for 
a long time now with automatic_addressbook) is not shown here in Roundcube for 
most users. For my test-user it is shown, but only after I created one CardDAV 
address book, Google Contacts in this case. The options for collected addresses 
in the settings are marked and active.

II. Steps to reproduce the Issue: I have no idea, here it can be shown with any 
standard user here in our installation.
```

Original issue reported on code.google.com by `msebald1...@googlemail.com` on 2 Nov 2012 at 1:18

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.7
```

Original comment by `roland.l...@gmail.com` on 2 Nov 2012 at 3:51
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Works! :-) Thanks for the great and fast help!
```

Original comment by `msebald1...@googlemail.com` on 2 Nov 2012 at 4:33
